### ISSUE TITLE
Let throttle take an action and function with no arguments

### ISSUE THREAD

**Issue by Aldaviva** [Aldaviva/ThrottleDebounce#1]
Affects version 1.0.1

**Comment by Aldaviva** [comment]
Resolved. Now you can throttle actions with no arguments.
```cs
DebouncedAction throttledAction = Throttler.Throttle(() => {
    //do something
}, TimeSpan.FromSeconds(1));
```
Same with functions that take no arguments.
```cs
DebouncedFunc<bool> throttledFunction = Throttler.Throttle(() => {
    return true;
}, TimeSpan.FromSeconds(1));
```

**Comment by Aldaviva** [comment]
Released in [1.0.2](https://www.nuget.org/packages/ThrottleDebounce/1.0.2).
